### PR TITLE
Change the description of xml data model

### PR DESCRIPTION
**Pull request by KavinduZoysa**
## Purpose
$subject

### CODE REVIEW

File: examples/xml-data-model/xml_data_model.description [hunk 1]
```diff
@@ -1,8 +1,8 @@
 // An `xml` value is a sequence representing the parsed content of an XML element. <br></br>
 // An `xml` value has four kinds of items.
 // <ul>
-// <li>`element`, `processing instruction` and `comment` item correspond 1:1 to XML infoset items</li>
-// <li>`text` item corresponds to one or more Character Information Items</li>
+// <li>The `element`, `processing instruction` and `comment` singletons correspond directly to information items in the XML Information Set</li>
```
Comment (praneesha) [inline examples/xml-data-model/xml_data_model.description:4]:
```suggestion
// <li>The `element`, `processing instruction`, and `comment` singletons correspond directly to the information items in the XML information set</li>
```

Comment (KavinduZoysa) [reply examples/xml-data-model/xml_data_model.description]: Fixed

File: examples/xml-data-model/xml_data_model.description [hunk 2]
```diff
@@ -1,8 +1,8 @@
 // An `xml` value is a sequence representing the parsed content of an XML element. <br></br>
 // An `xml` value has four kinds of items.
 // <ul>
-// <li>`element`, `processing instruction` and `comment` item correspond 1:1 to XML infoset items</li>
-// <li>`text` item corresponds to one or more Character Information Items</li>
+// <li>The `element`, `processing instruction`, and `comment` singletons correspond directly to the information items in the XML information set</li>
```
Comment (MaryamZi) [inline examples/xml-data-model/xml_data_model.description:4]:
```suggestion
// <li>The element, processing instruction, and comment singletons correspond directly to the items in the XML information set</li>
```

File: examples/xml-data-model/xml_data_model.description [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
+// <li>`text` item corresponds to one or more character information items</li>
```
Comment (MaryamZi) [inline examples/xml-data-model/xml_data_model.description:5]:
```suggestion
// <li>The text item corresponds to one or more character information items</li>
```

File: examples/xml-data-model/xml_data_model.description (hunk 2 above)
Comment (KavinduZoysa) [reply examples/xml-data-model/xml_data_model.description]: Fixed

File: examples/xml-data-model/xml_data_model.description (hunk 3 above)
Comment (KavinduZoysa) [reply examples/xml-data-model/xml_data_model.description]: Fixed

**Review by pubudu91 — APPROVED** [review]

**Pull request merged**